### PR TITLE
Add Flags from the New England States

### PR DESCRIPTION
**Pull request by LordBarkBread**
#### Summary
Content "Add Flags from the New England States"

#### Purpose of change
Gives representation to the six states within the New England region

#### Describe the solution
Add a state flag item group
Add 6 total variants to the state flag item

#### Testing
image
Starting the game without error
Spawn in state flag without error
See the variations of the state flags

#### Additional context
Need to add it as a loot item in certain places

### CODE REVIEW

**Comment by Zireael07** [comment]
I think state names should be capitalized?

**Comment by LordBarkBread** [comment]
> I think state names should be capitalized?

Fixed.

File: data/json/items/armor/cloaks.json [hunk 1]
```diff
@@ -148,6 +148,56 @@
       }
     ]
   },
+  {
+    "id": "state_flag",
+    "repairs_like": "sheet",
+    "type": "ARMOR",
+    "name": { "str": "state flag" },
+    "description": "A large white flag with a shield featuring an American Indian and a star which represents the state of Massachusetts.",
```
Comment (Maleclypse) [inline data/json/items/armor/cloaks.json:156]:
```suggestion
    "description": "A large white flag with a shield featuring an American Native and a star which represents the state of Massachusetts.",
```
editing to make it consistent with your text below.

File: data/json/items/armor/cloaks.json [hunk 2]
```diff
@@ -148,6 +148,56 @@
...
+      {
+        "id": "vermont_flag",
+        "name": { "str": "Vermont flag" },
+        "description": "A large blue flag featuring a shield with wreaths on both sides, a pine tree in a green field along with a cow and bales of wheat reside within the shield representing Vermont.",
+        "weight": 5
+      },
+      {
+        "id": "connecticut_flag",
+        "name": { "str": "Connecticut flag" },
+        "description": "A large blue flag featuring a white baroque shield with three grapevines, each bearing three bunches of purple grapes on a field of royal blue, below the shield reads “Qui Transtulit Sustinet”, which is latin for “He who transplanted sustains” which represents the state of Connecticut.",
```
Comment (TheShadowFerret) [inline data/json/items/armor/cloaks.json:190]:
```suggestion
        "description": "A large blue flag featuring a white baroque shield with three grapevines, each bearing three bunches of purple grapes on a field of royal blue, below the shield reads “Qui Transtulit Sustinet”, which is Latin for “He who transplanted sustains” which represents the state of Connecticut.",
```

File: data/json/items/armor/cloaks.json [hunk 3]
```diff
@@ -148,6 +148,56 @@
...
+      {
+        "id": "massachusetts_flag",
+        "name": { "str": "Massachusetts flag" },
+        "description": "A large white flag with a shield featuring an American Native and a star which represents the state of Massachusetts.",
+        "weight": 5
+      },
+      {
+        "id": "maine_flag",
+        "name": { "str": "Maine flag" },
+        "description": "A large blue flag featuring a moose and pine tree emblem on a shield supported by a farmer and a sailor, above is the north star and the latin motto “Dirigo” (“I direct”), below the shield reads out the state of “Maine”.",
```
Comment (TheShadowFerret) [inline data/json/items/armor/cloaks.json:172]:
```suggestion
        "description": "A large blue flag featuring a moose and pine tree emblem on a shield supported by a farmer and a sailor, above is the north star and the Latin motto “Dirigo” (“I direct”), below the shield reads out the state of “Maine”.",
```

**Pull request merged**